e2e/runner/zeta.go

```diff
@@ -31,6 +34,30 @@ func (r *E2ERunner) WaitForBlocks(n int64) {
 	err = retry.DoWithBackoff(call, boWithMaxRetries)
 	require.NoError(r, err, "failed to wait for %d blocks", n)
 }
+
+// WaitForTSSGeneration waits for a specific number of TSS to be generated
+// The parameter n is the number of TSS to wait for
+func (r *E2ERunner) WaitForTSSGeneration(tssNumber int64) {
+	call := func() error {
+		return retry.Retry(r.checkNumberOfTssGenerated(tssNumber))
+	}
+	bo := backoff.NewConstantBackOff(time.Second * 5)
+	boWithMaxRetries := backoff.WithMaxRetries(bo, 10)
+	err := retry.DoWithBackoff(call, boWithMaxRetries)
+	require.NoError(r, err, "failed to wait for %d tss generation", tssNumber)
+}
+
+func (r *E2ERunner) checkNumberOfTssGenerated(tssNumber int64) error {
```

Comment:
```suggestion
func (r *E2ERunner) checkNumberOfTSSGenerated(tssNumber int64) error {
```
We really need to always remember using fully uppercase for acronym, this is small detail but still help for code clarity